HousingRepairsOnlineApi/Controllers/RepairController.cs

```diff
@@ -23,7 +31,17 @@ public async Task<IActionResult> SaveRepair([FromBody] RepairRequest repairReque
             try
             {
                 var result = await saveRepairRequestUseCase.Execute(repairRequest);
-
+                switch (repairRequest?.ContactDetails?.Type)
+                {
+                    case "email":
```

Comment:
`"email"` and `"sms"` are [Magic Strings](https://en.wikipedia.org/wiki/Magic_string) and should be avoided.
Instead, create a class with constants for each of the valid communication channels and replace the literal uses with the new constants throughout the code base.